test/unit/dsl/objects_test.rb

```diff
@@ -472,10 +472,10 @@
       control.id = "tag.control.id"
       control.to_ruby.must_equal '
 control "tag.control.id" do
-  tag "key": "value"
-  tag "key2\'": "value\'"
-  tag "key3\"": "value\""
-  tag "key4": ["a", "b"]
+  tag key: "value"
+  tag key2\': "value\'"
+  tag key3": "value\""
+  tag key4: ["a", "b"]
```

Comment:
This is NOT valid ruby for `key2` and `key3`. So either this fix isn't comprehensive enough, or the test is overkill because we never intend to support such keys/values.